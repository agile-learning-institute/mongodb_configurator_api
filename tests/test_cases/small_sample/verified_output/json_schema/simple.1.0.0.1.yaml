description: A simple collection for testing
type: object
properties:
  _id:
    description: The unique identifier for the media
    type: string
    pattern: "^[0-9a-fA-F]{24}$"
  name:
    description: The name of the document
    type: string
    pattern: "^[a-zA-Z0-9_-]{4,40}$"
  status:
    description: The current status of the document
    type: string
    enum: ["active", "archived"]
additionalProperties: false
required:
  - _id
  - status  