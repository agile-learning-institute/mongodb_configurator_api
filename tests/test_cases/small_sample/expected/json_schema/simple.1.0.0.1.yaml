type: object
properties:
  _id:
    type: string
    pattern: "^[0-9a-fA-F]{24}$"
  name:
    type: string
    pattern: "^[a-zA-Z0-9_-]{4,40}$"
  status:
    type: string
    enum: ["active", "archived"]
additionalProperties: false
required:
  - _id
  - status  